core/videolib/convertor.py

```diff
@@ -3,20 +3,21 @@
 
 
 def concat_all_ts(video_file: str) -> None:
-    """
+    """Concat all ts chucks one by one.
+
     Parameters
     ----------
     video_file: str
         The file_name by which to save the concatenated .ts files
     """
-
     subprocess.Popen(["ffmpeg", "-f", "concat", "-safe", "0", "-i", "ts_list.txt",
                       "-c", "copy", f"{video_file}.ts"], stdout=subprocess.DEVNULL).wait()
     os.unlink("ts_list.txt")
 
 
 def convert_video(video_input: str, video_output: str) -> None:
-    """
+    """Convert video from ts to mp4.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)